perf/roles/trex/tasks/main.yml

```diff
@@ -9,29 +9,47 @@
       - python3
       - rdma-core-devel
       - zlib-devel
+      - driverctl
```

Comment:
Now that you are not building trex anymore, you should be removing all **build** dependencies and replace them with **runtime** dependencies.

Reply:
Updated